src/content/docs/mobile-monitoring/new-relic-mobile/maintenance/customizing-your-mobile-app-settings.mdx

```diff
@@ -79,6 +76,16 @@ Available settings include:
       </td>
     </tr>
 
+    <tr>
+      <td>
+        Block or allow hostnames, create rules for status codes, create aliases for hostnames
```

Comment:
We could add a link to this table column to another newly created page, which would be dedicated to these settings, or to a querying page?